### PR TITLE
fix(cmd/mv): dst filename error

### PR DESCRIPTION
**Pull request by chenminjian**
#5963 

License: MIT
Signed-off-by: chenminjian <727180553@qq.com>

### CODE REVIEW

**Comment by magik6k** [comment]
(go-mfs PR - https://github.com/ipfs/go-mfs/pull/62)

**Comment by Stebalien** [comment]
Dropped the gx update as mfs has already been updated. Kept the test.

File: test/sharness/t0250-files-api.sh [hunk 1]
```diff
@@ -521,7 +521,11 @@ test_files_api() {
 
   # test mv
   test_expect_success "can mv dir $EXTRA" '
-    ipfs files mv /cats/this/is /cats/
+    ipfs files mv /cats/this/is /
```
Comment (Stebalien) [inline test/sharness/t0250-files-api.sh]: We should test moving to _both_ `/cats/` and `/cats`. Maybe move to `/cats/`, then to `/`, then to `/cats`?

Comment (chenminjian) [reply test/sharness/t0250-files-api.sh]: OK, I have done it.

**Pull request merged**